### ISSUE TITLE
Cannot configure container image build preferences

### ISSUE THREAD

**Issue by DecisionSystems** [GoogleCloudPlatform/cloud-code-vscode#283]
(Please ensure you are running the latest version of Cloud Cloud IntelliJ with _Help > Check for Updates_.)

(screenshots are often helpful)

### Expected Behavior
Following instructions on this page:  https://cloud.google.com/code/docs/intellij/image-build-settings 
I have an existing application.

### Actual Behavior
Nothing happens when I do this:

Navigate to the Run/Debug configurations dialog on the top taskbar or the top-level main menu: Run > Edit Configurations... > Develop on Kubernetes or Run on Kubernetes and choose the Build/Deploy tab.

### Aditional Information

 - Running MiniKube / Hyper-V for storing Docker and MiniKube objects

- IDE type (e.g. IntelliJ, Pycharm): Visual Studio Code
- IDE version: Version 1.48.0
- Cloud Code version (Settings > Plugins > Cloud Code):  Version 1.5.0
- Cloud SDK (Settings > Cloud Code > Cloud SDK)
  - Are you allowing the plugin to manage the Cloud SDK: 
  - Version of the Cloud SDK:
- Skaffold version (If you are using Kubernetes features; Settings > Cloud Code > Kubernetes): Yes
- Operating System: Windows 10 Enterprise build 2004

_Feel free to deviate from this template as needed, especially if you are submitting a feature request._

**Comment by matthewmichihara** [comment]
@DecisionSystems does that error notification appear whenever you try to select `Run on Kubernetes`?

**Comment by DecisionSystems** [comment]
Not that I know of.  I did not display in the foreground.  I only noticed it sometime later as a result of a notification showing in the bell icon on the lower right of the screen.  I have other warnings about slow activation startups but Cloud Code is the only one showing an uncaught error message.

**Comment by DecisionSystems** [comment]
I clicked- report the issue.  Here is the output.  Plz advice on how to resolve ASAP.  This is holding up our on-going development.
Thanks,

- Issue Type: `Bug`
- Extension Name: `cloudcode`
- Extension Version: `1.5.0`
- OS Version: `Windows_NT x64 10.0.19041`
- VSCode version: `1.48.0`

:warning: We have written the needed data into your clipboard. Please paste! :warning:

```json 
{
	"messages": [],
	"activationTimes": {
		"codeLoadingTime": 373,
		"activateCallTime": 3,
		"activateResolvedTime": 23977,
		"activationReason": {
			"startup": true,
			"extensionId": {
				"value": "googlecloudtools.cloudcode",
				"_lower": "googlecloudtools.cloudcode"
			},
			"activationEvent": "*"
		}
	},
	"runtimeErrors": [
		{
			"name": "Error",
			"message": "Failed to configure docker authentication, Jib builds may fail to push to Google Container Registries as a result.  Error: Traceback (most recent call last):\r\n  File \"C:\\Users\\rkarabush\\AppData\\Local\\cloud-code\\installer\\google-cloud-sdk\\bin\\..\\lib\\gcloud.py\", line 20, in <module>\r\n    from __future__ import absolute_import\r\nImportError: cannot import name absolute_import\r\n"
		}
	]
}
```

**Comment by redborian** [comment]
It looks like `gcloud auth configure-docker -q` command is failing on initialize. Here are a few things that you can do to help us troubleshoot this faster:
1. Are you opted into auto-managing dependencies? (check `"cloudcode.autoDependencies"` setting in your VS Code setting)
1. Do you have a GCP account? Can you check if you have logged in by running `gcloud auth list` command from your terminal? If not logged in, can you login using `gcloud auth login` command from your terminal?
1. Now try running `gcloud auth configure-docker -q` command from your terminal.

**Comment by DecisionSystems** [comment]
1. Cloudcode: Auto Dependencies is ON.
2. Do we have a GCP account? NO
    Checked and gcloud command is not recognized in terminal
    Tried logging in; Could not as command is not recognized.

3. Frankly, I've never logged into GCloud.  We are developing an app that hopefully will run on many, including Google.  From what I remember reading, this is supposed to work without having a GCloud account, Yes?

Thanks,

**Comment by redborian** [comment]
Thanks for the quick turn-around, @DecisionSystems. If auto-manage dependencies is turned ON, gcloud will be present here: `%localappdata%\cloud-code\installer\google-cloud-sdk\bin`.
Our plugin will work across all cloud providers. We are looking into this issue and will fix it soon. As an interim measure, try an older version of our extension. [Instructions here](https://stackoverflow.com/questions/42626065/vs-code-rollback-extension-install-specific-extension-version/53755378)

**Comment by DecisionSystems** [comment]
Hopefully you can respond just as quickly with a fix. Don't make me go back to trying to get the MS K8s plugin working :-)

I tried version 1.4 - Progress. Got a prompt for the configuration but no respond when I clicked to create one.
I tried version 1.3 - No mention of debug on Kubernetes, which is what I'm trying to do.

SOL

Thanks,

**Comment by redborian** [comment]
Can you verify that even though you got the "runtime error", Cloud Code still loads the extension? You can verify this by checking whether the Kubernetes Explorer/GKE explorer tree view loads.

**Comment by DecisionSystems** [comment]
I looked and I don't think it does.  I only see </> Cloud Code at the bottom of the VS Code screen and can't seem to locate the explorer tree anywhere.  Of course, I've never seen it before but I obtained images about what it looks like and can't locate it anywhere.  How about a quick webconference soon?  It might save everyone some time :-)

**Comment by jmah-cc** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-vscode-internal](https://github.com/GoogleCloudPlatform/cloud-code-vscode-internal/issues/3046)

**Comment by redborian** [comment]
@DecisionSystems will it be possible for you to repro in the latest version of Cloud Code? It will help us isolate the problem further.

**Comment by DecisionSystems** [comment]
I can however since I reported this, I broke down and signed-up for a google cloud account 😊  Unfortunately, I’ve yet to get this working, even with the account, since I’ve had other issues setting up the organization within the setup.

Does it make sense for me to still try?  I can remove some of your libraries if need be.  Let me know?

Thanks,

Reid Karabush

[Decision Systems Incorporated]<http://www.decisionsystems.com/>

5 Revere Drive, Suite 200
Northbrook, Illinois 60062, USA
office:    1.847.433.0876 ext. 101
mobile:  1.847.722.7870
www.decisionsystems.com<http://www.decisionsystems.com/>
[view_my_linkedin_profile]<https://www.linkedin.com/in/reidkarabush>
